frontend/src/services/fetch.service.ts

```diff
@@ -0,0 +1,10 @@
+import { getAsyncItem } from './async-storage.service'
```

Comment:
Dit is de authenticated fetch die we nu kunnen gebruiken in plaats van de fetch en axios. Als jullie liever iets anders hebben, let me know.